packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -48,7 +50,27 @@ typedef GestureDragCancelCallback = void Function();
 ///  * [PanGestureRecognizer], for drags that are not locked to a single axis.
 abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   /// Initialize the object.
-  DragGestureRecognizer({ Object debugOwner }) : super(debugOwner: debugOwner);
+  DragGestureRecognizer({
+    Object debugOwner,
+    this.dragStartBehavior = DragStartBehavior.start,
+  }) : super(debugOwner: debugOwner);
+
+  /// Configure the behavior of offsets sent to [onStart].
```

Comment:
(this comment should be on line 85, but github doesn't allow me to comment there) Can you add something to the doc for `onStart` that references this property here, so people who look there find this?